config/config.go

```diff
@@ -3,8 +3,8 @@ package config
 import (
 	"io/ioutil"
 
-	"github.com/brancz/locutus/rollout"
-	"github.com/brancz/locutus/trigger/types"
+	"github.com/rollandf/locutus/rollout"
```

Comment:
you want to have the repo in your gopath but under my user, then you don't need thi

Reply:
Done